src/control/server/instance_drpc.go

```diff
@@ -29,28 +29,31 @@ var (
 	errEngineNotReady = errors.New("engine not ready yet")
 )
 
-func (ei *EngineInstance) setDrpcClient(c drpc.DomainSocketClient) {
+func (ei *EngineInstance) setDrpcSocket(sock string) {
 	ei.Lock()
 	defer ei.Unlock()
-	ei._drpcClient = c
+	ei._drpcSocket = sock
```

Comment:
nit: don't see a need for the underscore variable prefix

Reply:
Based on a comment on the struct definition, the reason this was initially done was to mark variables that need to be behind a lock. As this variable subject to the same limitations _drpcClient originally was, I kept the underscore.